src/pages/home/components/ActionBar/ActionBar.module.scss

```diff
@@ -37,11 +46,15 @@
 
     border-radius: 50%;
 
-    .arrowIcon {
-      color: $primaryWhite;
+    &:disabled {
```

Comment:
Creio que seria interessante colocar um cursor not allowed quando estivesse disabled